Automattic/jetpack
# Payment blocks: Fixed toolbar control edge cases

Fixes #23633

#### Changes proposed in this Pull Request:

* Replaced the "Add new subscription" in the Customizer context.
* Close the pre-publish/post-publish when the user clicks on the "Add new subscription" option and the panels are open.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Apply xxx-code on your sandbox
* Use Jetpack Beta to switch to this branch.

**Test closing the publish panel**
* Add a new Payment button block
* Click on the post button in order to open the post/pre-publish panel.
* Click on the Add new subscription from the toolbar control
* Notice that the pre-publish panel is closed and the Product subscription panel is opened.

**Replace the Add new subscription menu item with a link**
- Add a new Payment button block
- Click on the "Add new subscription" menu link
- You should notice that you'll be redirected in a new tab to /earn/payment-plans page.
